ambuda/views/blog.py

```diff
@@ -113,7 +117,7 @@ def delete_post(slug):
             session.delete(post_)
             session.commit()
 
-            flash(f"Deleted post {slug}")
+            flash(f"Deleted post {slug}", "success")
```

Comment:
any reason deleted post is not wrapped with _l()?

Reply:
Actually, are blog posts going to be localized as well? If not, ignore my comment.

Reply:
There's some context on this at the top of this module. But generally, no -- this text is for admins and site maintainers, so there's no need to localize this for the foreseeable future.